slsim/Deflectors/elliptical_lens_galaxies.py

```diff
@@ -59,7 +65,31 @@ def __init__(
         if "n_sersic" not in column_names:
             galaxy_list["n_sersic"] = -np.ones(n)
         if "gamma_pl" not in column_names:
-            galaxy_list["gamma_pl"] = np.ones(n) * 2
+            if gamma_pl is None:
```

Comment:
is there a way where we perhaps do not have to duplicate the code in two classes but instead use one routine/definition for it?